src/main/java/nus/climods/logic/parser/parameters/optional/UserFlagParser.java

```diff
@@ -0,0 +1,12 @@
+package nus.climods.logic.parser.parameters.optional;
+
+import nus.climods.logic.parser.exceptions.ParseException;
+
+/**
+ * Class for parsing user flag
+ */
+public class UserFlagParser extends OptionalArgumentsParser {
```

Comment:
Is this parser really necessary? I forsee we can just have a parser for each command and will just include the user flag `Option` in it along with the necessary `CommandLineParser` and etc.